schema.graphql

```diff
@@ -56,15 +88,26 @@ enum EventName {
 }
 
 type EscrowEvent @entity {
+  "hash transaction of the escrow"
   id: ID!
+  "escrow contract address"
   contractAddress: Bytes!
+  "is the reference data used to track the escrow event"
   reference: Bytes!
+  "smart contract that triggers when a payment is disbursed"
```

Comment:
Please change to "the Escrow where this event occurred"

Reply:
Corrected